src/corona_hakab_model/medical_state.py

```diff
@@ -9,35 +9,20 @@ class MedicalState(State, ABC):
     test_willingness: float
     detectable: bool
 
-    def val(self):
-        return self.agent_count
-
-
-class SusceptibleState(MedicalState, ABC):
-    def __init__(self, *args, test_willingness: float, **kwargs):
+    def __init__(self, *args, **kwargs):
+        self.susceptible = kwargs.pop('susceptible')
+        self.contagiousness = kwargs.pop('contagiousness')
+        self.test_willingness = kwargs.pop('test_willingness')
+        self.detectable = kwargs.pop('detectable')
```

Comment:
make actual keyword arguments, not this pop stuff

Reply:
Sure... but I'm not sure that this is smart since ultimately it means that MedicalState will call it's State superclass with `kwargs` that State does not expect to receive (breaking any future type-checking that we decide to add). This seems like undesirable behaviour.